docs/en/_sidebar.md

```diff
@@ -17,6 +17,8 @@
         - [Workload Type](/en/cue/workload-type.md)
         - [Trait](/en/cue/trait.md)
         - [Advanced Features](/en/cue/status.md)
+      - HELM
+        - [Chart As Capability](/en/helm/chart-as-capability.md)
```

Comment:
```suggestion
        - [Helm Chart Basic](/en/helm/chart-as-capability.md)
```

Reply:
Separate the above doc into two parts. One is deploy helm chart as workload, one is using existing traits with helm.

Reply:
got it.